eopdev/account.hcl

```diff
@@ -8,4 +8,6 @@ locals {
       created_outside_terraform = true
     }
   }
+
+  budget_monitoring_notification_email_addresses = ["e778fce5.greaterwellington.onmicrosoft.com@au.teams.ms", "eop+eopdev_budget_notifications@gw.govt.nz"]
```

Comment:
Only minor, but can these be extracted to variables somehow? Also, is it worth including an actual persons email address, like you/Sam/Chris?